patches/third_party-tflite-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
+index f11b0a89eca81b0d69803f1eb79fa80be7bf00e1..4be477c7291f4d246008398dbe2c959e3ac26146 100644
+--- a/third_party/tflite/BUILD.gn
++++ b/third_party/tflite/BUILD.gn
+@@ -201,6 +201,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
+     ":tflite_benchmark",
+     "//services/webnn/*",
+     "//third_party/mediapipe/*",
++    "//third_party/tflite_support/*",
```

Comment:
`visibility += brave_tflite_builtin_op_resolver_visibility`

Reply:
fixed in https://github.com/brave/brave-core/pull/24953/commits/9d4da565d2b06599bea4ef9e6c1c365f7963d3f4